### PR TITLE
Update reinforcement learning tutorial to sync with master

### PR DESCRIPTION
**Pull request by puhuk**
Master : https://github.com/pytorch/ignite/tree/master/examples/reinforcement_learning
Website : https://pytorch-ignite.ai/tutorials/intermediate/03-reinforcement-learning/

### CODE REVIEW

**Comment by puhuk** [comment]
@vfdev-5 Here is the link :)
https://drive.google.com/file/d/1SfQEqm24XWSUQwaBtdt2t-_fHlokH19j/view?usp=sharing

**Comment by vfdev-5** [comment]
@puhuk thanks, looks good !

**Comment by vfdev-5** [comment]
Can you also please add instruction to install deps for colab as text.

**Comment by puhuk** [comment]
@vfdev-5 I updated to the colab link. Is that ok like this?

```
On Colab
We need additional dependencies to render the environment on Google Colab.
Xvfb : X virtual framebuffer, for display
pyvirtualdisplay : Python wrapper for Xvfb
python-opengl : Python binding to OpenGL
pygame, moviepy, imageio_ffmpeg : Render the output to video file
```
https://drive.google.com/file/d/1SfQEqm24XWSUQwaBtdt2t-_fHlokH19j/view?usp=sharing

**Comment by vfdev-5** [comment]
@puhuk I mean in the notebook for those who would like to run the tutorial on colab. So add new cell with text:


> On Colab we need to install additional packages:
> ```
!apt-get install -y xvfb python-opengl
!pip install pyvirtualdisplay
!pip install pygame moviepy==1.0.3 imageio_ffmpeg
```

File: tutorials/intermediate/03-reinforcement-learning.ipynb [hunk 1]
```diff
@@ -107,7 +107,8 @@
    "outputs": [],
    "source": [
     "!apt-get install -y xvfb python-opengl\n",
-    "!pip install pyvirtualdisplay"
+    "!pip install pyvirtualdisplay\n",
+    "!pip install pygame moviepy==1.0.3 imageio_ffmpeg"
```
Comment (vfdev-5) [inline tutorials/intermediate/03-reinforcement-learning.ipynb:111]:
```suggestion
    "!pip install --upgrade pygame moviepy"
```

**Pull request merged**